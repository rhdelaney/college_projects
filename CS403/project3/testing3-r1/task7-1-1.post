(println "trying (twinPrimes 9)")
(stream-display 81 (twinPrimes 9))
(println)
