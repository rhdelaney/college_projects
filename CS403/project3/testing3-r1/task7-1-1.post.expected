trying (twinPrimes 9)
((2 3) (2 5) (2 7) (2 11) (3 5) (3 7) (3 11) (5 7) (5 11) (5 13) (7 11) (7 13) (11 13) (11 17) (11 19) (13 17) (13 19) (17 19) (17 23) (19 23) (23 29) (23 31) (29 31) (29 37) (31 37) (37 41) (37 43) (41 43) (41 47) (43 47) (47 53) (53 59) (53 61) (59 61) (59 67) (61 67) (67 71) (67 73) (71 73) (71 79) (73 79) (79 83) (83 89) (89 97) (97 101) (97 103) (101 103) (101 107) (101 109) (103 107) (103 109) (107 109) (107 113) (109 113) (127 131) (131 137) (131 139) (137 139) (149 151) (149 157) (151 157) (157 163) (163 167) (167 173) (173 179) (173 181) (179 181) (191 193) (191 197) (191 199) (193 197) (193 199) (197 199) (223 227) (223 229) (227 229) (227 233) (229 233) (233 239) (233 241) (239 241) ...)
